config/buildspec_core.yml

```diff
@@ -0,0 +1,34 @@
+# TODO add different buildspecs and end to end sagemaker tests for each framework and version.
+# TODO set up code coverage
+# Build Spec for AWS CodeBuild CI which will run for every PR.
+
+version: 0.2
+env:
+  variables:
+    ## TODO: expand these environment variables once sagemaker tests are added for every framework and version.
+    run_pytest_core: "disable"
+phases:
+  install:
+    commands:
+        -  . config/change_branch.sh
+        - su && apt-get update
+        - apt-get install sudo -qq -o=Dpkg::Use-Pty=0 # silence output: https://askubuntu.com/a/668859/724247
+        - sudo apt-get update -qq -o=Dpkg::Use-Pty=0
+        - sudo apt-get install unzip -qq -o=Dpkg::Use-Pty=0
+        - pip install --upgrade pip==19.3.1
+        - pip install -q pytest==6.1.2 pytest-cov==2.10.1 wheel==0.35.1 pytest-html==3.0.0 pre-commit==2.6.0 awscli==1.18.199
+
+  pre_build:
+    commands:
+      - cd $CODEBUILD_SRC_DIR && pre-commit install && pre-commit run --all-files
+
+  build:
+    commands:
+      - cd $CODEBUILD_SRC_DIR  && python setup.py bdist_wheel --universal
+      - cd $CODEBUILD_SRC_DIR  && chmod +x config/tests.sh && PYTHONPATH=. && ./config/tests.sh  && mkdir -p upload/$CURRENT_COMMIT_PATH/wheels && cp ./dist/*.whl upload/$CURRENT_COMMIT_PATH/wheels && cd ..
```

Comment:
will you also release a source distribution?

Reply:
Your question seems unrelated to this change. But yes, when we release to PyPi, we will also release a source distribution.